src/main/java/app/persistence/OrderMapper.java

```diff
@@ -92,10 +112,12 @@ public static Order placeOrderInDB(User currentUser, Order order, ConnectionPool
                     return orderWithId;
 
                 } else {
+                    System.out.println("Order not inserted");
                     throw new DatabaseException("Order not inserted");
                 }
             }
         } catch (SQLException e) {
+            System.out.println("SQL Exception in placeOrderInDB");
```

Comment:
pritnline shuld proboly thow the error to the controller so it can give the msg to the frontend